.buildkite/pipelines/flaky_tests/runner.js

```diff
@@ -183,6 +183,18 @@ for (const testSuite of testSuites) {
         concurrency_group: UUID,
         concurrency_method: 'eager',
       });
+    case 'cypress':
```

Comment:
Based on the changes above we should probably define the three keys here and pull the `label` from the groups.json maybe?

Reply:
Made the changes! Thanks for the feedback